components/02-elements/swatch/_swatch.scss

```diff
@@ -38,47 +29,41 @@ $swatch__border-color--color--active  : $white !default;
         display: flex;
         justify-content: center;
         align-items: center;
-        height: $swatch__width;
-        width: $swatch__width;
-        border: $swatch__border;
-        background: $swatch__background;
-        margin-right: $swatch__margin;
-        margin-bottom: $swatch__margin;
+        min-width: $swatch__option-sizing;
+        min-height: $swatch__option-sizing;
+        margin: $swatch__option-margin;
+        border: $swatch__option-border;
+        background: $swatch__option-background;
+        color: $swatch__option-color;
         transition: $swatch__transition;
-        color: $swatch__label-color;
         cursor: pointer;
 
         &:hover {
-            border-color: $swatch__border-color-hover;
-            color: $swatch__label-color-hover;
+            border-color: $swatch__option-border-color-hover;
         }
 
         &:last-child {
             margin-right: 0;
         }
 
-        &--active {
-            background: $swatch__background--active;
-            border-color: $swatch__border-color--active;
-            color: $swatch__label-color--active;
-
-            &:hover {
-                color: $swatch__label-color--active;
-            }
+        @include mq($screen-m) {
+            min-width: $swatch__option-sizing-screen-m;
+            min-height: $swatch__option-sizing-screen-m;
         }
 
-        &--color {
-            border-color: $swatch__border-color--color;
-            outline: $swatch__outline--color;
+        &--active {
+            border-color: $swatch__option-border-color--active;
+        }
 
-            &:hover {
-                border-color: $swatch__border-color-hover--color;
-                outline-color: $swatch__outline-color-hover--color;
+        &--attributes {
```

Comment:
so how it is finally - we should add modifiers in BEM on the beginning of block or on the end? ;)

Reply:
but it's fine. It's modifier in `swatch__option` class and it's after all mq, and hover/focus statements, then I have modifiers and in the end I have other classes (in this case there is no other classes)

Reply:
ok, my mistake, it's modifier from element 👯